src/app/components/play-round/play-round.component.scss

```
@@ -18,10 +18,9 @@
 
 .interact-button {
   position: absolute;
-  left:50%;
-  transform: translate(-50%, 0);
-  bottom: 5%;
-  text-transform: uppercase;
+  right: 10px;
+  bottom: 10px;
```

```suggestion
  right: 16px;
  bottom: 16px;
```

https://material.io/components/buttons-floating-action-button#specs

---

Ah, nice, thanks!